src/Tool/WrapperInterface.php

```diff
@@ -63,10 +63,11 @@ public function getMetadata();
      * Get the object identifier, single or composite.
      *
      * @param bool $single
+     * @param bool $flatten
      *
      * @return array|mixed Array if a composite value, otherwise a single scalar
      */
-    public function getIdentifier($single = true);
+    public function getIdentifier($single = true/*, $flatten = false*/);
```

Comment:
Please, add a `@todo` docblock explaining what to do in the next major release.
BTW, this change also requires an entry in the changelog.